create-infra.yml

```diff
@@ -137,3 +137,11 @@
           ansible.builtin.include_role:
             name: sushy_emulator
             tasks_from: verify.yml
+
+        - name: Set permissions on /home/zuul/ci-framework-data on controller-0
+          ansible.builtin.file:
+            path: "/home/zuul/ci-framework-data"
```

Comment:
is it not "{{ cifmw_basedir }}" ?

Reply:
I am not sure it is available, so I decided to just hard-code - but on second thought it is better to use cifmw_basedir and a default value. I'll update the PR.

Reply:
Done.